templates/rhpam70-trial-ephemeral.yaml

```diff
@@ -104,10 +104,6 @@ parameters:
   name: MAVEN_REPO_USERNAME
   required: true
   value: mavenUser
-- displayName: Maven repository password
```

Comment:
Why not leaving the maven user password? just add the generated as well

Reply:
See my comment below.